src/Common/src/Interop/Linux/procfs/Interop.ProcFsStat.cs

```diff
@@ -18,13 +18,15 @@ internal static partial class procfs
         private const string CmdLineFileName = "/cmdline";
         private const string StatFileName = "/stat";
         private const string MapsFileName = "/maps";
+        private const string StatusFileName = "/status";
         private const string FileDescriptorDirectoryName = "/fd/";
         private const string TaskDirectoryName = "/task/";
 
         internal const string SelfExeFilePath = RootPath + "self" + ExeFileName;
         internal const string SelfCmdLineFilePath = RootPath + "self" + CmdLineFileName;
         internal const string ProcStatFilePath = RootPath + "stat";
 
+        private static char[] Delimiters => new char[] { ':', ' ', '\t' };
```

Comment:
This is allocating a new array every time Delimiters is accessed.

Reply:
Removed.